hopp/simulation/technologies/layout/wind_layout.py

```diff
@@ -73,16 +80,32 @@ class WindBoundaryGridParameters(BaseClass):
     #TODO: rename to border_spacing_ratio?
     border_spacing: float = field(default=0.0)
     #TODO: rename to border_offset_ratio?
-    border_offset: float = field(default=0.0, validator=range_val(0.0, 1.0)) 
-    border_spacing_m: Optional[float] = field(default=None)
-    min_spacing_m: Optional[float] = field(default=None)
+    border_offset: float = field(default = 0.0, validator = range_val(0.0, 1.0)) 
+    border_spacing_m: Optional[float] = field(default = None)
+
+    min_spacing_m: Optional[float] = field(default = 0.0)
+    min_spacing_D: Optional[float] = field(default = 2.0)
+    max_spacing_m: Optional[float] = field(default = 2e6)
+    max_spacing_D: Optional[float] = field(default = 20.0)
```

Comment:
Could you add `validator=validators.instance_of(float, type(None))` to each of these? Just a note that it's `type(None)` because it doesn't know to do an `is None` under the hood. Line 6 will also need to have `validators` as an additional import from attrs.